tests/integration/test_cli_benchmark_analytics_undo_display_utils.py

```
@@ -0,0 +1,2619 @@
+        assert degraded is False
+        assert reasons == []
+
+    def test_summarize_suite_classifications_mixed_effective_suites(self) -> None:
+        from file_organizer.cli.benchmark import (
+            _SuiteExecutionClassification,
+            _summarize_suite_classifications,
+        )
+
+        c1 = _SuiteExecutionClassification(effective_suite="io", degraded=False)
+        c2 = _SuiteExecutionClassification(effective_suite="text", degraded=False)
+        eff, degraded, reasons = _summarize_suite_classifications(
```

Ruff (0.15.9)

[warning] Unpacked variable `degraded` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)